packages/components/tsconfig.json

```diff
@@ -16,7 +17,7 @@
     "skipLibCheck": true,
     "strict": true,
     "suppressImplicitAnyIndexErrors": true,
-    "target": "es5"
+    "target": "es6"
```

Comment:
Our Babel build was targeting the last 2 versions of each browser, which doesn't meet Edu's browser support policy.

Unfortunately tsc doesn't have the ability to target specific browsers, so we give up that ability.

es6/es2015 is more broad than we need, but I think it's reasonable for the design system to output it.

Reply:
LP browser support for FF is "firefox >= 52" but according to https://caniuse.com/es6, partial support for ff v52, not sure how accurate that is tho

Reply:
Good catch. The .browserslist in LP is actually out of date. Sometime we should update it to what's at https://go.czi.team/edu-browser-support.

For Firefox we actually want to support >= 86